cicd/pipelines/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+include::_attributes/common-attributes.adoc[]
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+In {pipelines-shortname}, pods that result from a pipeline run or task run use a non-privileged security context by default. The default service account for these pods is `pipeline`, and the security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is similar to the `anyuid` SCC, but with minor differences as defined in the following YAML specification:
```

Comment:
> use a non-privileged security context by default
this might be a bit misleading for customers - on one hand this might mean that the security context being used is not the "privileged" security context, or it might mean that the security context being used is not privileged by nature and hence pretty restrictive, which is not the case as the `pipelines-scc` is pretty relaxed and privileged.

i suggest removing this line entirely to avoid confusion:
> pods that result from a pipeline run or task run use a non-privileged security context by default.